backend/tracim_backend/models/data.py

```diff
@@ -1422,12 +1434,17 @@ def format_path(cls, url_template: str, content: "Content") -> str:
         cid = content.content_id
         return url_template.format(wid=wid, fid=fid, ctype=ctype, cid=cid)
 
-    def copy(self, parent):
-        cpy_content = Content()
-        for rev in self.revisions:
-            cpy_rev = ContentRevisionRO.copy(rev, parent)
-            cpy_content.revisions.append(cpy_rev)
-        return cpy_content
+    @property
+    def recursive_tree_revision(self):
```

Comment:
do not use a property, it's a real "function" here.

`recursive_tree_revision` => `get_tree_revisions`

Reply:
done

Reply:
@inkhey this is a plural form, so revision**s**. By the way, you are missing the typing.